Kconfig.dependencies

```diff
@@ -59,7 +59,6 @@ config SIDEWALK_CRYPTO
 	imply PSA_WANT_ALG_DETERMINISTIC_ECDSA
 	imply PSA_WANT_ALG_STREAM_CIPHER
 	imply PSA_WANT_ALG_CCM
-	imply PSA_WANT_ALG_GCM
```

Comment:
Protocol Stack 1.0, Document Revision A.1 Amazon Sidewalk Specification 4.1.1 Security model
 "Network data confidentiality and integrity are provided by the AES-GCM algorithm"